docs-source/content/userguide/managing-domains/domain-lifecycle/introspection.md

```diff
@@ -44,6 +44,21 @@ Set `introspectVersion` to a new value.
 
 As with `restartVersion`, the `introspectVersion` field has no required format; however, we recommend using a value likely to be unique such as a continually increasing number or a timestamp.
 
+Starting from operator 3.1.0, if a domain resource's `spec.introspectVersion` is set, each of the domain's WebLogic Server pods will have a label with the key `weblogic.IntrospectVersion` to indicate the `introspectVersion` that the pod is running at.
+
+```
+Name:           domain1-admin-server
+Namespace:      domain1-ns
+Priority:       0
+Labels:         weblogic.createdByOperator=true
+                weblogic.domainName=domain1
+                weblogic.domainUID=domain1
+                weblogic.introspectVersion=12345
+                weblogic.serverName=admin-server
+```
+
+When a domain's `spec.intrsopectVersion` is changed, the `weblogic.introspectVersion` label of each WebLogic Server pod is updated to the new `introspectVersion` value either when the operator restarts the pod, or when the operator determines that the pod does not need to be restarted.
```

Comment:
value either -> value, either (comma here)
the pod, or when -> the pod or when (no comma here)